systemd/systemd_test.go

```diff
@@ -1753,7 +1753,7 @@ func (s *SystemdTestSuite) TestPreseedModeRemoveMountUnitUnmounted(c *C) {
 
 	sysd := NewEmulationMode(dirs.GlobalRootDir)
 	mountUnit := makeMockMountUnit(c, mountDir)
-	symlinkPath := filepath.Join(dirs.SnapServicesDir, "multi-user.target.wants", filepath.Base(mountUnit))
+	symlinkPath := filepath.Join(dirs.SnapServicesDir, "snapd.mounts.target.wants", filepath.Base(mountUnit))
```

Comment:
should we also add a different unrelated .wants or .requires symlink to test the new code in emulation remove logic?

Reply:
I have added that in the test. I also made the emulation a bit more like systemd by removing empty `.wants`/`.requires` directories after removing symlinks.